src/aks-preview/azext_aks_preview/_validators.py

```diff
@@ -639,6 +639,14 @@ def validate_enable_custom_ca_trust(namespace):
                 '--enable_custom_ca_trust can only be set for Linux nodepools')
 
 
+def validate_disable_windows_outbound_nat(namespace):
+    """Validates disable_windows_outbound_nat can only be used on Windows."""
+    if namespace.disable_windows_outbound_nat:
+        if hasattr(namespace, 'os_type') and namespace.os_type != "Windows":
```

Comment:
Should handle case-insensitive comparation?

Reply:
Improved. Thank you.